eng/platform-matrix.yml

```diff
@@ -206,7 +206,7 @@ jobs:
           - Windows.81.Amd64
           - Windows.10.Amd64
           - Windows.10.Amd64.Core
-          - Windows.10.Nano.Amd64
+          - (Windows.Nano.1803.Amd64.Open)windows.10.amd64.serverrs4.open@mcr.microsoft.com/dotnet-buildtools/prereqs:nanoserver-1803-helix-amd64-05227e1-20190509225944
```

Comment:
Do we want to be using an open queue here on the internal builds?

Reply:
Definitely not, I missed that.  It will fail trying to send there;  just delete the .open.